qt/widgets/mplcpp/src/Colors.cpp

```diff
@@ -176,8 +179,11 @@ Python::Object SymLogNorm::tickLocator() const {
   // Create log transform with base=10
   auto transform = scaleModule().attr("SymmetricalLogTransform")(
       10, Python::Object(pyobj().attr("linthresh")), m_linscale);
+  std::vector<float> subsVector(10);
+  std::iota(subsVector.begin(), subsVector.end(), 1);
```

Comment:
I think the MSVC build warnings are due to an integer `1` being use to initialize a `vector<float>`. Using `1.f` should do it.